src/pam_hbac_obj.c

```diff
@@ -16,6 +16,9 @@
 */
 
 #define _GNU_SOURCE
+#ifndef __size_t               // to avoid not compiling on hpux
+  #define __size_t size_t
+#endif
```

Comment:
This change seems to break compilation on Linux. I will try to find a solution that would hopefully work on Linux as well, but could you please paste the compilation error you saw on HP-UX?